### ISSUE TITLE
Installation on latest OS X fails

### ISSUE THREAD

**Issue by gavinhughes** [Rudd-O/ledgerhelpers#3]
running `sudo python setup.py install`

```
...
running install_data
creating /usr/share/applications
error: could not create '/usr/share/applications': Operation not permitted
```

**Comment by Rudd-O** [comment]
This can be fixed by adding an exception to the `setup.py` file, whereby the files in `/usr/share/applications` do not get installed at all if you are installing on Mac OS X.

Question: How does one detect that one is running the install routine in Mac OS X?  This will help me speed up making a branch for you to test.

**Comment by gavinhughes** [comment]
This is due to OSX's [System Integrity Protection](https://support.apple.com/en-us/HT204899).  The best way to install ledgerhelpers on Mac is using [Homebrew](http://brew.sh/), just like ledger itself.

**Comment by Rudd-O** [comment]
Can you provide instructions so that I can add them to the README.md ?  Happy to do so.

**Comment by gavinhughes** [comment]
The instructions are [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md), also the [formula for ledger itself](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/ledger.rb).

Making the formula is something a maintainer/contributor does. The end user can then install with `brew install ledgerhelpers`.

**Comment by Rudd-O** [comment]
I am very sorry but I cannot make the formula myself, as I have a grand total of zero Mac OS X boxes, and I don't plan to buy one.  However, if someone contributes a formula either upstream or as a pull request to this project, _I will merge it in a heartbeat_ and update the install instructions.

**Comment by themiurgo** [comment]
@Rudd-O How about using entry-points instead? http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point

**Comment by Rudd-O** [comment]
I still think it is a good idea to add an exception for those data files in Mac OS X.  This needs to be done independent of whether there is Homebrew or not.  I am happy to take patches / pull requests.  Thanks.

**Comment by ThatIsAPseudo** [comment]
Shouldn't this issue be closed since PR #7 was merged, as well as removing the last paragraph regarding macOS installation from the Readme ?

Imho another issue should be created (or this one should be renamed) to track the creation of a Homebrew formula.

**Comment by Rudd-O** [comment]
Agreed.  Will close this ticket.  It would be more than welcome if you opened a ticket tracking Homebrew formula creation.
